packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -9,14 +9,15 @@ import 'package:flutter/foundation.dart';
 import 'package:path/path.dart';
 
 import '../charts/treemap.dart';
-import 'vm_treemap.dart';
+import 'temp/vm_treemap.dart';
```

Comment:
nit:`tmp` is more standard naming for a temporary directory

Reply:
Done